esx_service/vmdk_ops.py

```diff
@@ -376,8 +376,10 @@ def vol_info(vol_meta, vol_size_info, datastore):
     vinfo[CAPACITY][ALLOCATED] = vol_size_info[ALLOCATED]
     vinfo[LOCATION] = datastore
 
-    if kv.ATTACHED_VM_NAME in vol_meta:
-       vinfo[ATTACHED_TO_VM] = vol_meta[kv.ATTACHED_VM_NAME]
+    if kv.ATTACHED_VM_UUID in vol_meta:
+       vm = findVmByUuid(vol_meta[kv.ATTACHED_VM_UUID])
+       if vm:
+          vinfo[ATTACHED_TO_VM] = vm.config.name
```

Comment:
With this change, "kv.ATTACHED_VM_NAME" is redundant. Should we remove "ATTACHED_VM_NAME" since it also doesn't affect backward compatibility?